app/api/server/v1/teams.ts

```diff
@@ -189,16 +191,23 @@ API.v1.addRoute('teams.listRoomsOfUser', { authRequired: true }, {
 API.v1.addRoute('teams.members', { authRequired: true }, {
 	get() {
 		const { offset, count } = this.getPaginationItems();
-		const { teamId, teamName } = this.queryParams;
-		const { query } = this.parseJsonQuery();
+		const { teamId, teamName, status, username, name } = this.queryParams;
+
+		// check(status, [String]);
```

Comment:
If you won't use this, then remove :)

Reply:
I was waiting for #21414 so that I could implement the typechecks here in the same way. Done now 😄